.github/workflows/soundness.yml

```
@@ -143,7 +143,7 @@ jobs:
     if: ${{ inputs.format_check_enabled }}
     runs-on: ubuntu-latest
     container:
-      image: swiftlang/swift:nightly-6.0-jammy
+      image: swift:6.0.1-jammy
```

```suggestion
      image: swift:6.0-jammy
```

This will allow us to pick up dot releases without needing new PR.

---

Updated